src/components/TransactionsInterpreter/helpers/getDenominatedValue.ts

```diff
@@ -0,0 +1,28 @@
+import { UITransactionType } from './types';
+import { networkConfigSelector } from 'reduxStore/selectors';
+import { store } from 'reduxStore/store';
+import { denominate } from 'utils';
+import { DenominationConfig } from '../TransactionsInterpreter';
+
+export function getDenominatedValue(
+  transaction: UITransactionType,
+  denominationConfig: DenominationConfig
+) {
+  const networkConfig = networkConfigSelector(store.getState());
+
+  const value = transaction.value;
+  const { showLastNonZeroDecimal = false } = denominationConfig;
+  const decimals = denominationConfig.decimals
```

Comment:
you can just do ?? to fallback to the default option, or maybe even set this in the destructuring of arguments